src-java/testing/test-library/src/main/java/org/openkilda/testing/service/grpc/GrpcServiceImpl.java

```diff
@@ -63,13 +66,15 @@ public SwitchInfoStatus getSwitchStatus(String switchAddress) {
     }
 
     @Override
+    @Retryable
```

Comment:
unfortunately `@Retryable` won't work with negative tests
we need to add some exclude

Reply:
added conditions